plugins/headlamp-plugin/bin/headlamp-plugin.js

```diff
@@ -231,6 +233,71 @@ function extract(pluginPackagesPath, outputPlugins) {
   return 0;
 }
 
+/**
```

Comment:
headlamp/plugins/headlamp-plugin/template/package.json is where you'd add it so it's added to peoples packages.

When you do `npm run pack` should it output in the current directory? Like `npm pack` does?

Reply:
yeah, I was wondering about what the package.json script should be named. I think 2 packs will be confusing?